guacamole/src/main/frontend/src/translations/ca.json

```diff
@@ -163,6 +183,80 @@
 
     },
 
+    "IMPORT": {
+
+        "ACTION_ACKNOWLEDGE"        : "@:APP.ACTION_ACKNOWLEDGE",
+        "ACTION_BROWSE"             : "Navega pels fitxers",
+        "ACTION_CANCEL"             : "@:APP.ACTION_CANCEL",
+        "ACTION_CLEAR"              : "@:APP.ACTION_CLEAR",
+        "ACTION_VIEW_FORMAT_HELP"   : "Veure l'ajuda de format",
+        "ACTION_IMPORT"             : "@:APP.ACTION_IMPORT",
+        "ACTION_IMPORT_CONNECTIONS" : "Importa connexions",
+
+        "DIALOG_HEADER_ERROR"   : "@:APP.DIALOG_HEADER_ERROR",
```

Comment:
In this section, I think you can leave out the `@` references to other sections, unless you're seeing issues where this isn't translating things correctly?

Reply:
Removed all the @APP lines